src/components/common/Header.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// Header element that takes in a string as a prop and displays it as the title
+const Header = ({ title }) => {
+  return (
+    <header className="storysquad-header">
+      <h1 className="header-title">{title}</h1>
+      {/* Background image of the header */}
+      <img src="/images/header-image.png" alt="underwater shark cityscape" />
+    </header>
+  );
+};
+
+export default Header;
+
+Header.propTypes = {
```

Comment:
I can see kind of what it is doing but I would still like to know more about the library.